test-runner/src/main/scala/com/typesafe/spark/test/mesos/DCOSSpec.scala

```diff
@@ -0,0 +1,22 @@
+package com.typesafe.spark.test.mesos
+
+import org.scalatest.FunSuite
+import org.scalatest.concurrent.TimeLimitedTests
+
+import com.typesafe.spark.test.mesos.framework.runners.RoleConfigInfo
+
+class DCOSSpec (
```

Comment:
Ok what is the reason of removing it (@skyluc) ? Yes that is a good idea if somethign should not run on DCOS, ideally everything should run everywhere but anyway...